src/it/scala/io/iohk/ethereum/txExecTest/ForksTest.scala

```diff
@@ -24,6 +24,8 @@ class ForksTest extends FlatSpec with Matchers {
     override val daoForkBlockNumber: BigInt = 10000
     override val daoForkBlockHash: ByteString = ByteString("unused")
     override val daoForkBlockTotalDifficulty: BigInt = 1000000
+    override val difficultyBombPauseBlockNumber: BigInt = Long.MaxValue
+    override val difficultyBombContinueBlockNumber: BigInt = Long.MaxValue
```

Comment:
@rtkaczyk Since difficulty bomb starts growing only after block 200 000 we would have to dump at least that many blocks for integration test data. I think it doesn't make sense to do that. I could adjust the algorithm for tests but what's the point of doing integrations test where the function to be tested is not exactly the same? Let's stick to unit tests with this one